styles/lazy-styles.css

```diff
@@ -17,7 +17,7 @@
 }
 
 @font-face {
-  font-family: ProximaNova;
+  font-family: ProximaNovaLight;
```

Comment:
revert this change. We are using font-weight to control different font-families. see https://www.w3schools.com/cssref/pr_font_weight.php